packages/core/src/routes/verification-code.ts

```diff
@@ -21,6 +21,7 @@ export default function verificationCodeRoutes<T extends AuthedRouter>(
     '/verification-codes',
     koaGuard({
       body: requestVerificationCodePayloadGuard,
+      status: [204, 400, 404, 422],
```

Comment:
is there any test case for 404 and 422 error?

Reply:
The 422 code is the error code for the DB insertion error.
404 errors will not happen in theory. （removed)